src/script/connection/ConnectionRepository.ts

```diff
@@ -124,7 +124,9 @@ export class ConnectionRepository {
       await this.userRepository.refreshUser(connectionEntity.userId);
       // Get conversation related to connection and set its type to 1:1
       // This case is important when the 'user.connection' event arrives after the 'conversation.member-join' event: https://wearezeta.atlassian.net/browse/SQCORE-348
-      amplify.publish(WebAppEvents.CONVERSATION.MAP_CONNECTION, connectionEntity);
+
+      const isWebSocketEvent = source === EventRepository.SOURCE.WEB_SOCKET;
+      amplify.publish(WebAppEvents.CONVERSATION.MAP_CONNECTION, connectionEntity, isWebSocketEvent);
```

Comment:
how about passing the `source` there and have the logic of `=== SOURCE.WEB_SOCKET` in the `mapConnection` method?

Reply:
@atomrc it works for me https://github.com/wireapp/wire-webapp/pull/15743/commits/d5398be40d4fdfb884839dea83a85a4957660fb0 ;)